.github/workflows/vscode-deploy-artifacts.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy VSCode Artifacts
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build_python_artifacts:
+    name: Build Python Artifacts
+    uses: jondavid-black/AaC/.github/workflows/python-build-artifacts.yml@20a5f29a2b77d80f68d0edab64c8d5e7760204ec
```

Comment:
I detest how Github Actions references work, but there may be a way to ignore specifying the commit hash. I found this "the called workflow is from the same commit as the caller workflow." in the docs https://docs.github.com/en/actions/using-workflows/reusing-workflows. 

At the moment, I don't know of a better alternative to using the commit hashes; I recall there being an issue trying to avoid using the commit hashes. 

As for falling out of sync, it's not really a big deal since the pipeline will just be an older version of the jobs/steps, but will still use the up-to-date repo to create artifacts.